Makefile

```diff
@@ -29,115 +29,90 @@
 ###############################################################################
 include config.mk
 
-.PHONY : all
-.PHONY : install-dev
 .PHONY : virtualenv
-.PHONY : install-workflow-deps
 .PHONY : install-merlin
-.PHONY : clean-output
-.PHONY : clean-docs
-.PHONY : clean-release
-.PHONY : clean-py
-.PHONY : clean
-.PHONY : release
+.PHONY : install-workflow-deps
+.PHONY : install-merlin-dev
 .PHONY : unit-tests
-.PHONY : cli-tests
+.PHONY : e2e-tests
 .PHONY : tests
 .PHONY : fix-style
 .PHONY : check-style
+.PHONY : check-push
 .PHONY : check-camel-case
 .PHONY : checks
 .PHONY : reqlist
-.PHONY : check-variables
-
-
-all: install-dev install-merlin install-workflow-deps
-
-
-# install requirements
-install-dev: virtualenv
-	$(PIP) install -r requirements/dev.txt
-
-
-check-variables:
-	- echo MAX_LINE_LENGTH $(MAX_LINE_LENGTH)
-
+.PHONY : release
+.PHONY : clean-release
+.PHONY : clean-output
+.PHONY : clean-docs
+.PHONY : clean-py
+.PHONY : clean
 
-# this only works outside the venv
+# this only works outside the venv - if run from inside a custom venv, or any target that depends on this,
+# you will break your venv.
 virtualenv:
-	$(PYTHON) -m venv $(VENV) --prompt $(PENV) --system-site-packages
-	$(PIP) install --upgrade pip
-
-
-install-workflow-deps:
-	$(PIP) install -r $(WKFW)feature_demo/requirements.txt
-
-
-install-merlin:
-	$(PIP) install -e .
-
+	(
+	    $(PYTHON) -m venv $(VENV) --prompt $(PENV) --system-site-packages; \
+	    . $(VENV)/bin/activate; \
+	    $(PIP) install --upgrade pip; \
+	    $(PIP) install -r requirements/release.txt; \
+	)
 
-# remove python bytecode files
-clean-py:
-	-find $(MRLN) -name "*.py[cod]" -exec rm -f {} \;
-	-find $(MRLN) -name "__pycache__" -type d -exec rm -rf {} \;
 
-
-# remove all studies/ directories
-clean-output:
-	-find $(MRLN) -name "studies*" -type d -exec rm -rf {} \;
-	-find . -maxdepth 1 -name "studies*" -type d -exec rm -rf {} \;
-	-find . -maxdepth 1 -name "merlin.log" -type f -exec rm -rf {} \;
+# install merlin into the virtual environment
+install-merlin: virtualenv
+	( \
+	   . $(VENV)/bin/activate; \
+	   $(PIP) install -e .; \
+	   merlin config; \
+	)
 
 
-# remove doc build files
-clean-docs:
-	rm -rf $(DOCS)/build
+install-workflow-deps: virtualenv install-merlin
+	( \
+	   . $(VENV)/bin/activate; \
+	   merlin example feature_demo; \
```

Comment:
i don't think you need to do this, b/c won't $(WKFW) point directly to where the example is in the source file?

Reply:
Oh, yes! I just noticed the disparity between the Make targets and the CI and was just synching them up - didn't check the config vars that closely when I pulled them out :D